core/oauth/application/metric.go

```diff
@@ -0,0 +1,12 @@
+package application
+
+import (
+	"go.opencensus.io/stats"
+)
+
+var (
+	// LoginFailedCount counts the failed login attempts
+	LoginFailedCount = stats.Int64("flamingo/oauth/login_failed", "Count of failed login attempts", stats.UnitDimensionless)
```

Comment:
Does this need to be public?

Reply:
I could move these from `application` to `interface`, otherwise yes.

Reply:
Okay actually either way it has to be exported.